internal/security/secretstore/tls/cert.go

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package tls
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func createCA(commonName, certPath, keyPath string) error {
+	caSubject := fmt.Sprintf("/C=UK/O=IOTech/CN=%s", commonName)
```

Comment:
```suggestion
	caSubject := fmt.Sprintf("/C=UK/O=EdgeXFoundry/CN=%s", commonName)
```
use a neutral name

Reply:
tls related comments will be addressed in the following PR.